### PR TITLE
Changed deprecated JSON to json

### PR DESCRIPTION
**Pull request by dineshbhosale**
Replace deprecated `JSON` in favor of `json`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,5 +3,7 @@
 .packages
 .pub/
 build/
+.idea
```
Comment (kevmoo) [inline .gitignore]: We explicitly don't ignore IDE files. Please put this in your `.gitignore_global`

File: .gitignore [hunk 2]
```diff
@@ -3,5 +3,7 @@
...
+pubspec.lock
```
Comment (kevmoo) [inline .gitignore]: We explicitly DO checkin the pubspec.lock file – to track which dependencies we use

File: angular_gallery/lib/builder/gallery_lib_builder.dart [hunk 3]
```diff
@@ -92,7 +92,7 @@ class GalleryLibBuilder extends Builder {
       if (!await buildStep.canRead(gallerySectionSummaryId)) continue;
 
       final summaries =
-          JSON.decode(await buildStep.readAsString(gallerySectionSummaryId));
+          json.decode(await buildStep.readAsString(gallerySectionSummaryId));
```
Comment (kevmoo) [inline angular_gallery/lib/builder/gallery_lib_builder.dart]: use `jsonDecode` instead

**Comment by dineshbhosale** [comment]
I make a new PR

**Pull request closed without merging**